pkg/translator/azure/property_names.go

```diff
@@ -0,0 +1,78 @@
+package azure
+
+var mappings = map[string]map[string]string{
+	"common": {},
+	"FrontDoorAccessLog": {
+		"trackingReference":   "",
+		"httpMethod":          "http.request.method",
+		"httpVersion":         "http.request.version",
```

Comment:
I believe [network.protocol.version](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/network.md) would be the right match